.circleci/config.yml

```diff
@@ -342,6 +350,5 @@ workflows:
       - coverage:
           requires:
             - unittest-27
-            - smoke-docker-json
             - smoke-docker-syslog
-            - smoke-k8s
```

Comment:
I would just comment out the lines for `smoke-docker-json` and `smoke-k8s` with a TODO to re-enable them when they are passing again.